src/jupyter_contrib_nbextensions/nbextensions/toc2/main.js

```diff
@@ -47,74 +42,49 @@ define([
                 'navigate_num': '#000000',
               },
               collapse_to_match_collapsible_headings: false,
-        skip_h1_title: false,
 }
+    // default values for per-notebook configurable parameters
+    var metadata_settings = {
+        nav_menu: {},
+        number_sections: true,
+        sideBar: true,
+        skip_h1_title: false,
```

Comment:
To me, it's per-notebook, as whether it makes sense depends on the notebook content?